sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 ## Release History
 
-### 1.9.0-beta.1 (Unreleased)
+### 2.0.0-beta.1 (Unreleased)
```

Comment:
You increased the major version - I think .Net did as well - because this PR contains a breaking change. Please call out what exactly is (when) breaking in the #### Breaking Changes section of the changelog

Reply:
Added